app/src/test/java/org/mozilla/fenix/library/history/HistoryComponentTest.kt

```diff
@@ -39,22 +39,39 @@ class HistoryComponentTest {
     }
 
     @Test
-    fun `add and remove one history item normally`() {
-        val historyItem = HistoryItem(123, "Mozilla", "http://mozilla.org", 0)
+    fun `select two items for removal, then deselect one, then select it again`() {
```

Comment:
@colintheshots I updated this test and added a new one for the new behavior (deselecting all items triggers normal mode) 😄